v2/googlecloud-to-googlecloud/src/main/java/com/google/cloud/teleport/v2/templates/spannerchangestreamstobigquery/FailsafeModJsonToTableRowTransformer.java

```diff
@@ -205,17 +210,37 @@ public void processElement(ProcessContext context) {
         }
       }
 
-      private TableRow modJsonStringToTableRow(String modJsonString) throws Exception {
```

Comment:
any particular reason to change this function from private to default (no access modifier)?

Reply:
Good call, thank you. I was experimenting with testing this and seeing if I could instantiate this on its own but required a lot of coupled unnecessary infrastructure. I forgot to revert it back to private.